packages/browsers/src/browser-data/chromedriver.ts

```diff
@@ -54,17 +57,21 @@ export function relativeExecutablePath(
   switch (platform) {
     case BrowserPlatform.MAC:
     case BrowserPlatform.MAC_ARM:
+      return path.join('chromedriver-' + folder(platform), 'chromedriver');
     case BrowserPlatform.LINUX:
-      return 'chromedriver';
+      return path.join('chromedriver-linux64', 'chromedriver');
     case BrowserPlatform.WIN32:
     case BrowserPlatform.WIN64:
-      return 'chromedriver.exe';
+      return path.join('chromedriver-' + folder(platform), 'chromedriver.exe');
   }
 }
 export async function resolveBuildId(
-  _channel: 'latest' = 'latest'
+  channel: ChromeReleaseChannel
 ): Promise<string> {
-  return await getText(
-    new URL(`https://chromedriver.storage.googleapis.com/LATEST_RELEASE`)
-  );
+  if (channel === ChromeReleaseChannel.STABLE) {
```

Comment:
Why the special case here?
We are downloading exclusively from CtF bucket, could there be version miss match this way?

Reply:
a left-over, removed!